test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/EmptyAttributeTagHelpers.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@addtaghelper "something"
+
+<div>
+    <input type= checked=""class="" />
+    <p age=''>
+        <input type=""checked= class="" />
```

Comment:
this line contains an error (no value for `checked`).  why does code get generated?